src/plugins/mission_raw/mission_import.cpp

```diff
@@ -110,6 +123,122 @@ MissionImport::import_mission(const Json::Value& root)
     return {mission_items};
 }
 
+static float set_float(const Json::Value& val)
+{
+    return val.isNull() ? NAN : val.asFloat();
+}
+
+static float set_int32(const Json::Value& val)
+{
+    return static_cast<int32_t>(val.isNull() ? 0 : (std::round(val.asDouble() * 1e7)));
+}
```

Comment:
I would have these as `static` private class member functions instead, just for taste.